src/torrent_info.cpp

```diff
@@ -1094,7 +1094,6 @@ namespace {
 			int const num_nodes = merkle_num_nodes(num_leafs);
 			m_merkle_first_leaf = num_nodes - num_leafs;
 			m_merkle_tree.resize(num_nodes);
-			std::memset(m_merkle_tree.data(), 0, aux::numeric_cast<std::size_t>(num_nodes * 20));
```

Comment:
is it not important that all the nodes in the merkle tree are cleared? I recognize that ``memset()`` isn't such a great way of clearing it, but now you're not clearing it at all

Reply:
I suppose if this is just called once, when `m_merkle_tree` is empty, it's fine. It will be cleared by `resize()`.

Reply:
Yes, I believe the call to `resize()` just above clears the elements.